### PR TITLE
Add .NET Azure Functions Worker SDK

### PR DESCRIPTION
**Pull request by mattjohnsonpint**
Adds the Azure Functions integration for .NET.

Docs PR: https://github.com/getsentry/sentry-docs/pull/7000

### CODE REVIEW

File: packages/nuget/Sentry.AzureFunctions.Worker/3.json
Comment (SeanFeldman) [inline packages/nuget/Sentry.AzureFunctions.Worker/3.json:1]: @mattjohnsonpint, so registration has to be done for each major _and_ minor on topo of that?

**Review by SeanFeldman — APPROVED** [review]
Thank you.
I didn't realize this information had to be registered somewhere.

**Comment by mattjohnsonpint** [comment]
@SeanFeldmen - I didn't expect you to know about this.  This sort of PR is only needed when first creating a new package.  Our release process manages it on new releases automatically after that.

File: packages/nuget/Sentry.AzureFunctions.Worker/3.json
Comment (mattjohnsonpint) [reply packages/nuget/Sentry.AzureFunctions.Worker/3.json:1]: They are symlinks, managed by the scripts in this repo.  Don't worry about it.

**Pull request merged**